### DISCUSSION TITLE
Feature Request: Add ability to copy repo settings to another repo

### DISCUSSION THREAD

**Discussion by daltonv** [community/community#31326] [Other Feature Feedback, Questions, & Ideas]
I manage more than a 100 repos in my organization and we often have the same repository settings between repos. ie branch protection rules, github pages, and access list. 

It would be great if I didn't have to manually set the same settings every time we make a new repository. Some option to just copy the settings (obviously excluding the repo name) from another repo in the org would be great. 

Alternatively the [template repo](https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/creating-a-template-repository) system could be upgraded to include the repo settings as part of the template. However, I wouldn't always want the same files and branches in a repo generated from a template.

**Comment by megamanics** [comment]
https://github.com/marketplace/actions/repo-settings-sync
